paths_cli/tests/compiling/test_engines.py

```
@@ -0,0 +1,74 @@
+import pytest
+import yaml
+import os
+
+from paths_cli.compiling.engines import *
+from paths_cli.compiling.errors import InputError
+import openpathsampling as paths
+
+from openpathsampling.engines import openmm as ops_openmm
+import mdtraj as md
+
+
+class TestOpenMMEngineBuilder(object):
+    def setup(self):
+        self.cwd = os.getcwd()
+        self.yml = "\n".join([
+            "type: openmm", "name: engine", "system: system.xml",
+            "integrator: integrator.xml", "topology: ad.pdb",
+            "n_steps_per_frame: 10", "n_frames_max: 10000"
+        ])
+        pass
+
+    def teardown(self):
+        os.chdir(self.cwd)
+
+    def _create_files(self, tmpdir):
+        mm = pytest.importorskip('simtk.openmm')
+        openmmtools = pytest.importorskip('openmmtools')
+        unit = pytest.importorskip('simtk.unit')
```

these will start to fail silently on newer openmm versions